### DISCUSSION TITLE
Lychee Demo site down

### DISCUSSION THREAD

**Discussion by MattPark** [LycheeOrg/Lychee#2513] [Bugs] [answered]
### Lychee version

unknown

### Which PHP version are you using?

PHP 8.3

### Detailed description of the problem

Lychee Demo site has been down a couple weeks.  I reported it in the community but looks like it wasn't the right place.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
I did try to solve it, I will have a look again.

**Reply by ildyria** [reply, marked as answer]
and we are back online: https://lychee-demo.fly.dev/landing

**Comment by MattPark** [comment]
Thanks!
